moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -68,10 +66,10 @@ export const PROJECT_LIST_VIEW_QUERY_CONFIG = {
       type: "String",
     },
     project_description: {
-      hidden: true,
       searchable: true,
       sortable: true,
       label: "Project description",
+      showInTable: false,
```

Comment:
the helper function loops through the column definitions looking for showInTable as true, so this isn't any different behavior, just making the definition explicit